plenum/test/restart/test_persist_pp.py

```diff
@@ -0,0 +1,134 @@
+from common.serializers.serialization import node_status_db_serializer
+from plenum.common.config_helper import PNodeConfigHelper
+from plenum.common.constants import LAST_SENT_PRE_PREPARE
+
+from plenum.test.test_node import TestNode, ensureElectionsDone, checkNodesConnected
+
+from plenum.test.helper import sdk_send_batches_of_random_and_check, \
+    checkViewNoForNodes, sdk_send_batches_of_random
+from plenum.test.restart.helper import restart_nodes
+
+nodeCount = 7
+batches_count = 3
+
+
+def get_primary_replicas(nodes):
+    primary_replicas = []
+    for instId in nodes[0].replicas.keys():
+        for idx, node in enumerate(nodes):
+            if node.replicas[instId].isPrimary:
+                assert instId == len(primary_replicas)
+                if node.replicas[instId].isMaster:
+                    primary_replicas.insert(0, node.replicas[instId])
+                else:
+                    primary_replicas.append(node.replicas[instId])
+                break
+    return primary_replicas
+
+
+def test_persist_last_pp(looper, txnPoolNodeSet, sdk_pool_handle, sdk_wallet_client, tconf):
+    primary_replicas = get_primary_replicas(txnPoolNodeSet)
+
+    seq_no_before_0 = primary_replicas[0].lastPrePrepareSeqNo
+    seq_no_before_1 = primary_replicas[1].lastPrePrepareSeqNo
+    seq_no_before_2 = primary_replicas[2].lastPrePrepareSeqNo
+
+    sdk_send_batches_of_random(looper, txnPoolNodeSet,
+                               sdk_pool_handle, sdk_wallet_client,
+                               batches_count, batches_count,
+                               timeout=tconf.Max3PCBatchWait)
```

Comment:
After sending requests we must wait until the 3PC-batches are ordered by all the replicas from `primary_replicas`.